src/init.js

```diff
@@ -0,0 +1,43 @@
+// Command to set up Cavy within a repo
+// `cavy init`
+//
+const indexContent = require('../templates/index.test.js');
+const specContent = require('../templates/exampleSpec.js');
+const { existsSync, writeFileSync, mkdirSync } = require('fs');
+
+DEFAULT_TEST_DIR = 'specs';
+DEFAULT_ENTRY_FILE = 'index.test.js';
+
+// Sets up Cavy entry index file and example spec.
+function init(args) {
+  console.log('cavy: Adding Cavy to your project...');
```

Comment:
It'd be nice to sanity check that we're running inside a react-native project. How does the React Native CLI do this?

```
» react-native foo
Command `foo` unrecognized. Make sure that you have run `npm install` and that you are inside a react-native project.
```